activesupport/lib/active_support/inflector/inflections.rb

```diff
@@ -44,7 +44,7 @@ def <<(*word)
         end
 
         def add(words)
-          words = words.flatten.map(&:downcase)
+          words = words.flatten.map(&:downcase).reject { |word| uncountable?(word) }
```

Comment:
Should probably just make `Uncountables` [inherit from](https://github.com/kenta-s/rails/blob/7b98ae7d0d8c6aee0f3414bf707ded65509cb5fe/activesupport/lib/active_support/inflector/inflections.rb#L31) `Set`.